### PR TITLE
Expand docs

### PR DESCRIPTION
**Pull request by elwinvaneede**
Please review the documentation site as a whole by looking at the deploy preview (see first comment by the Netlify bot).
Let me know if something is unclear, if you're missing something, or if there are grammatical errors.

### CODE REVIEW

File: src/getting-started/overview.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Graphite is funded and built by [Way2Web](https://www.way2web.nl), which means we build for the company’s business needs, but we’ve made it open source for anyone to use and contribute back to.
+
+It's available under the terms of the [MIT license](https://github.com/Way2Web/graphite-design-system/blob/master/LICENSE).
+
+## Target platform
+
+Graphite is mainly targeted at browser/desktop applications.
+
+We make sure our components are usable on mobile browsers with touch input. But for mobile apps we recommend using [Ionic](https://ionicframework.com/) (with [Capacitor](https://capacitorjs.com/)) instead of Graphite. Because of their [adaptive styling](https://ionicframework.com/docs/theming/platform-styles), Ionic apps look and feel at home on every device following the device guidelines while still offering some customization for branding. There are design kits available for [Sketch](https://ionicsketch.com/) & [Figma](https://ionicfigma.com/).
```
Comment (Tom-w2w) [inline src/getting-started/overview.md:27]: FYI: Don't know if we have to explain why to use a other system

**Review by Tom-w2w — APPROVED** [review]

**Review by panvakalo — APPROVED** [review]

File: src/getting-started/overview.md (hunk 1 above)
Comment (elwinvaneede) [reply src/getting-started/overview.md:27]: Yeah, agreed on second thought. I will reword this to something like we don't recommend using it for mobile apps because it doesn't adhere to the iOS Human Interface Guidelines & Android Material Design guidelines.

File: src/tokens/typography.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+[[toc]]
+
+<p class="intro">Typography tokens are used to maintain a consistent set of font styles throughout your app.</p>
+
+## Font family
+
+| Token              | Value                                                                                                 | Example                                                                    |
+| :----------------- | :---------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `--gr-font-family` | -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif | <div class="font-sans">The quick brown fox jumped over the lazy dog.</div> |
```
Comment (ricardoprins-w2w) [inline src/tokens/typography.md:15]: The original text contains `jumps` instead of `jumped`. Now you're missing the letter `s`

**Review by ricardoprins-w2w — APPROVED** [review]
Only did a code review: looks good to me! Only found a couple of textual changes for you to consider.

File: README.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ SOFTWARE.
 
 ### Nuxt Content
 
-The layout is based or inpired on [Nuxt Content](https://content.nuxtjs.org/).
+The layout is based or inspired on [Nuxt Content](https://content.nuxtjs.org/).
```
[the remark this answers is not in the document]
Comment (elwinvaneede) [reply README.md:69]: Yeah agreed, I will change this everywhere. Thanks.

File: src/tokens/typography.md (hunk 2 above)
Comment (elwinvaneede) [reply src/tokens/typography.md:15]: Sharp, thanks. Will fix this.
